include/swift/Sema/ConstraintSystem.h

```diff
@@ -2148,6 +2148,37 @@ struct ClosureIsolatedByPreconcurrency {
   bool operator()(const ClosureExpr *expr) const;
 };
 
+/// Describes a closure that is being processed by the constraint solver.
+struct ClosureInfo {
+private:
```

Comment:
Should this be a `class` instead?

Reply:
Good catch! I meant to change that and forgot :/